src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -56,12 +56,13 @@ const useGroupSetting = ({route}) => {
   }, [selectedChannel]);
 
   const submitData = async (withNavigation = true) => {
-    if (!changeImage || changeName) return navigation.goBack();
+    if (!changeImage && !changeName) return navigation.goBack();
     try {
       setLoadingUpdate(true);
       let body = {
         channel_id: route?.params?.id,
-        channel_name: groupName
+        channel_name: groupName,
+        channel_image: urlImage
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new property `channel_image` is being added to the `body` object regardless of whether `urlImage` is truthy or not. If `urlImage` is falsy (e.g., null, undefined, empty string), this could potentially cause issues on the server side. Consider moving the addition of `channel_image` inside the `if (urlImage)` block.
```diff
-        let body = {
-          channel_id: route?.params?.id,
-         channel_name: groupName,
-         channel_image: urlImage
-       };
-       if (urlImage) {
+        let body = {
+          channel_id: route?.params?.id,
+          channel_name: groupName,
+        };
+        if (urlImage) {
+          body.channel_image = urlImage;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
channel_image is used because to send the current image to the local db and api